src/jit/codegencommon.cpp

```diff
@@ -6722,11 +6722,11 @@ void CodeGen::genZeroInitFltRegs(const regMaskTP& initFltRegs, const regMaskTP&
                     inst_RV_RV(INS_vmov_i2f, reg, initReg, TYP_FLOAT, EA_4BYTE);
                 }
 #elif defined(_TARGET_XARCH_)
-                // Xorpd xmmreg, xmmreg is the fastest way to initialize a float register to
+                // Xorps xmmreg, xmmreg is the fastest way to initialize a float register to
                 // zero instead of moving constant 0.0f.  Though we just need to initialize just the 32-bits
                 // we will use xorpd to initialize 64-bits of the xmm register so that it can be
```

Comment:
Isn't the comment wrong now? It's still talking about using xorpd to init the register

Reply:
Right, I fixed only the first reference to xorpd. Thanks!

Reply:
I cleaned up the whole thing. Floats, doubles, 32 bit, 64 bits - it makes no difference, `xorps` is the "standard" way to zero out a SSE/AVX register. Sometimes `pxor` may be useful but `xorpd` is pointless.